src/preferences/dialog/dlgprefbeats.cpp

```diff
@@ -119,32 +96,14 @@ void DlgPrefBeats::fixedtempoEnabled(int i) {
     slotUpdate();
 }
 
-void DlgPrefBeats::offsetEnabled(int i) {
-    m_bOffsetEnabled = static_cast<bool>(i);
-    slotUpdate();
-}
-
-void DlgPrefBeats::minBpmRangeChanged(int value) {
-    m_minBpm = value;
-    slotUpdate();
-}
-
-void DlgPrefBeats::maxBpmRangeChanged(int value) {
-    m_maxBpm = value;
-    slotUpdate();
-}
-
 void DlgPrefBeats::slotUpdate() {
     checkBoxFixedTempo->setEnabled(m_bAnalyzerEnabled);
-    checkBoxOffsetCorr->setEnabled((m_bAnalyzerEnabled && m_bFixedTempoEnabled));
     comboBoxBeatPlugin->setEnabled(m_bAnalyzerEnabled);
     checkBoxAnalyzerEnabled->setChecked(m_bAnalyzerEnabled);
     // Fast analysis cannot be combined with non-constant tempo beatgrids.
     checkBoxFastAnalysis->setEnabled(m_bAnalyzerEnabled && m_bFixedTempoEnabled);
-    txtMaxBpm->setEnabled(m_bAnalyzerEnabled && m_bFixedTempoEnabled);
-    txtMinBpm->setEnabled(m_bAnalyzerEnabled && m_bFixedTempoEnabled);
     checkBoxReanalyse->setEnabled(m_bAnalyzerEnabled);
-    checkBoxReanalyseImported->setEnabled(m_bReanalyzeImported);
+    checkBoxReanalyseImported->setEnabled(m_bAnalyzerEnabled);
```

Comment:
Let's AE spelling consistently, i. e. "Reanaly**z**e"

Also, the checkbox name and the boolean variable name do not match, which might be confusing.